### PR TITLE
Feature/ui prototype build

### PR DESCRIPTION
**Pull request by ivanmrva**
mavenization and dockerization

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM nginx:1.15.2-alpine
+
+COPY css /usr/share/nginx/html/css
+COPY img /usr/share/nginx/html/img
+COPY js /usr/share/nginx/html/js
+COPY templates /usr/share/nginx/html/templates
+COPY index.html /usr/share/nginx/html/index.html
+COPY favicon.ico /usr/share/nginx/html/favicon.ico
```
Comment (99999989) [inline Dockerfile]: Please add a newline at end of file

File: README.md [hunk 2]
```diff
@@ -2,10 +2,30 @@
...
+
+## Build
+
+### With Maven
+
+1. Execute `./mvnw clean package`.
+
+### Without Maven
+1. Run `npm install` to install all dependencies.
+1. Run `gulp sass` to build the project.
```
Comment (99999989) [inline README.md]: The numbers should increase, I guess

File: README.md [hunk 3]
```diff
@@ -2,10 +2,30 @@
...
+
+## Run
+
+### With local httpserver
+1. Run `npm start` to start a local web server on port 8080.
+
+### As a docker image
+1. Build the project with _docker_ maven profile:  `./mvnw clean package -Pdocker`.
+1. Execute `docker run -d -p 8080:80 alvch-dockerv2-local.jfrog.io/alvch/online-services-ui-prototype:<project-version>`.
```
Comment (99999989) [inline README.md]: Also here, The numbers should increase

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+</project>
```
Comment (99999989) [inline pom.xml]: No newline at the end of file

File: README.md (hunk 2 above)
Comment (ivanmrva) [reply README.md]: image
-> it is working like this

this way you don't have to rewrite every line if you add a new point in the beginneing of the list later.

File: Dockerfile (hunk 1 above)
Comment (ivanmrva) [reply Dockerfile]: fixed

File: README.md (hunk 3 above)
Comment (ivanmrva) [reply README.md]: same as above comment

File: pom.xml (hunk 4 above)
Comment (ivanmrva) [reply pom.xml]: fixed

**Pull request merged**